docs/presentation/content/ContentInstancesOfSpecificClasses.md

```diff
@@ -98,7 +98,7 @@ Specifies an [ECExpression](./ECExpressions.md#instance-filter) for filtering in
 
 ### Attribute: `onlyIfNotHandled`
 
-Specifies whether this specification should be ignored if another specification was handled before as determined by rule and specification priorities. This provides a mechanism for defining a fallback specification.
+Specifies whether this specification should be ignored if another specification was *handled* before as determined by rule and specification priorities. This provides a mechanism for defining a fallback specification. **Note:** *handled* means that it was picked for evaluation, but doesn't mean it resulted in content being returned.
```

Comment:
Adding a note at the end is the least elegant fix. The documentation needs to be rewritten so as to not create any misconceptions in the first place. My attempt is as follows:

Specifies whether this specification should be ignored when another specification with higher priority exists in the same scope. This attribute is most useful for defining fallback specifications.

The above wording needs to be adapted to rules. I'm not convinced that "in the same scope" is the best phrasing but it's serviceable.

Reply:
I agree explaining this without a note would be better. I don't think the suggestion is completely correct, though:

> when another specification with higher priority exists in **the same scope**

I think it can be confusing for this case:
```
- rule1
  - spec1
- rule2
  - spec2 { onlyIfNotHandled: true }
```
If conditions of both `rule1` and `rule2` pass, then both `spec1` and `spec2` are used and the `onlyIfNotHandled` attribute on `spec2` takes effect. However, I wouldn't think that `spec2` is in the same scope as `spec1`.

What do you think about this:
```
Specifies whether this specification should be ignored when another specification with higher priority is used for the same request. This attribute is most useful for defining fallback specifications.
```

Reply:
The word "request" implies networking from the user's perspective thus I think we should avoid it.

I see that rules for `onlyIfNotHandled` eligibility are quite complex and deserve a separate learning article, but we need to somehow summarize them in the attribute description. Would you accept my suggestion if I exchanged the word "scope" for "context"?

Reply:
"Context" in this case means pretty much the same thing as "scope" and switching them doesn't help, IMO.

Another attempt:
```
Specifies whether this specification should be ignored when another specification with higher priority is used for creating the nodes. This attribute is most useful for defining fallback specifications.
```

Reply:
Your last suggestion is passable, but still lacks precision (as do my initial attempts). I want to know if you like the following more:

When `true`, the specification will take effect only when all other specifications with higher priority are ruled out. This attribute is most useful for defining fallback specifications.

Reply:
To me "ruled out" sounds like those specifications weren't used.
![image](https://user-images.githubusercontent.com/35135765/160536919-0fe4bae1-b316-4ba7-996b-e3c7b048603a.png)

Reply:
Is this not how that works? My understanding was, the higher priority rules and specifications are checked first, but if they all fail certain checks (e.g. `condition` evaluation), then specification with `onlyIfNotHandled: true` becomes eligible.

Reply:
Oh, sorry, I missed the key part that we're saying the spec **is** going to be used if the others are going to be ruled out. Yes, I believe your suggestion works best, I'll update the PR.

Reply:
Pushed. Please check.

Reply:
Looks good, thanks